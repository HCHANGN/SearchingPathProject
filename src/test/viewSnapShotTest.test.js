import React from "react";
import Graph from "../component/Graph";
import renderer from "react-test-renderer";

test("Test Graph Component", ()=>{
	let tree=renderer.create(<Graph />).toJSON();
	expect(tree).toMatchSnapshot();
});