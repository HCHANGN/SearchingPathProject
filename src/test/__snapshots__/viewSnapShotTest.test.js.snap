// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Graph Component 1`] = `
<div
  id="container"
>
  <div
    id="controlBar"
  >
    <div
      id="algoControl"
    >
      <div
        id="algoControlTitle"
      >
        Select Algorithm
      </div>
      <div
        id="algoControlSelectDiv"
      >
        <select
          id="algoSelect"
          name="algo"
          onChange={[Function]}
        >
          <option
            value="notYetSelect"
          >
            --Select your Algorithm--
          </option>
          <option
            value="DFS"
          >
            Depth First Search
          </option>
          <option
            value="BFS"
          >
            Breadth First Search
          </option>
          <option
            value="Dijk"
          >
            Dijkstra Algorithm
          </option>
          <option
            value="Astar"
          >
            A* Algorithm
          </option>
        </select>
      </div>
      <div
        id="calResult"
      >
        <div
          id="searchTimeResult"
        >
          <div>
            Actual search Time: 
            0.00
             ms
          </div>
        </div>
        <div
          id="visitedVertexResult"
        >
          <div>
            Total visited vertex: 
            0
          </div>
        </div>
      </div>
    </div>
    <div
      id="playControl"
    >
      <div
        id="playControlTitle"
      >
        Control panel
      </div>
      <div
        id="playControlBtn"
      >
        <div
          id="controlBtnSet"
        >
          <div
            onClick={[Function]}
          >
            <img
              id="playIcon"
              src="./img/playIcon.svg"
            />
          </div>
          <div
            onClick={[Function]}
          >
            <img
              id="pauseIcon"
              src="./img/pauseIcon.svg"
            />
          </div>
          <div
            onClick={[Function]}
          >
            <img
              id="restartIcon"
              src="./img/restartIcon.svg"
            />
          </div>
        </div>
        <div
          id="speedRange"
        >
          <div>
            -   Time Step   +
          </div>
          <input
            max="500"
            min="1"
            onChange={[Function]}
            type="range"
          />
        </div>
      </div>
    </div>
    <div
      id="addControl"
    >
      <div
        id="addControlTitle"
      >
        Select add mode
      </div>
      <div
        id="addModeSelectDiv"
      >
        <select
          id="modeSelect"
          name="addMode"
          onChange={[Function]}
        >
          <option
            value="wall"
          >
            Wall
          </option>
          <option
            value="weight"
          >
            Weight
          </option>
        </select>
      </div>
      <div
        id="addRandomBtn"
      >
        <button
          onClick={[Function]}
        >
          Add Random
        </button>
      </div>
    </div>
  </div>
  <div
    id="introIcon"
    onClick={[Function]}
  >
    ?
  </div>
  <div
    id="graphIntroContainer"
    style={
      Object {
        "display": "none",
      }
    }
  >
    <div
      id="graphIntroTitle"
    >
      Intro of Graph
    </div>
    <div
      id="startIntro"
    >
      <div
        id="startIntroBox"
      />
      <div
        id="startIntroText"
      >
         : Start
      </div>
    </div>
    <div
      id="endIntro"
    >
      <div
        id="endIntroBox"
      />
      <div
        id="endIntroText"
      >
         : End
      </div>
    </div>
    <div
      id="visitedIntro"
    >
      <div
        id="visitedIntroBox"
      />
      <div
        id="visitedIntroText"
      >
         : Visited
      </div>
    </div>
    <div
      id="wallIntro"
    >
      <div
        id="wallIntroBox"
      />
      <div
        id="wallIntroText"
      >
         : Wall
      </div>
    </div>
    <div
      id="weightIntro"
    >
      <div
        id="weightIntroBox"
      />
      <div
        id="weightIntroText"
      >
         : Weight
      </div>
    </div>
    <div
      id="SPIntro"
    >
      <div
        id="SPIntroBox"
      />
      <div
        id="SPIntroText"
      >
         : Shortest Path
      </div>
    </div>
  </div>
  <div
    id="graphContainer"
    onMouseDown={[Function]}
    onMouseMove={[Function]}
    onMouseOut={[Function]}
    onMouseUp={[Function]}
  >
    <div>
      <div
        className="startVertex"
        id="0_0"
      />
      <div
        className="vertex"
        id="0_1"
      />
      <div
        className="vertex"
        id="0_2"
      />
      <div
        className="vertex"
        id="0_3"
      />
      <div
        className="vertex"
        id="0_4"
      />
      <div
        className="vertex"
        id="0_5"
      />
      <div
        className="vertex"
        id="0_6"
      />
      <div
        className="vertex"
        id="0_7"
      />
      <div
        className="vertex"
        id="0_8"
      />
      <div
        className="vertex"
        id="0_9"
      />
      <div
        className="vertex"
        id="0_10"
      />
      <div
        className="vertex"
        id="0_11"
      />
      <div
        className="vertex"
        id="0_12"
      />
      <div
        className="vertex"
        id="0_13"
      />
      <div
        className="vertex"
        id="0_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="1_0"
      />
      <div
        className="vertex"
        id="1_1"
      />
      <div
        className="vertex"
        id="1_2"
      />
      <div
        className="vertex"
        id="1_3"
      />
      <div
        className="vertex"
        id="1_4"
      />
      <div
        className="vertex"
        id="1_5"
      />
      <div
        className="vertex"
        id="1_6"
      />
      <div
        className="vertex"
        id="1_7"
      />
      <div
        className="vertex"
        id="1_8"
      />
      <div
        className="vertex"
        id="1_9"
      />
      <div
        className="vertex"
        id="1_10"
      />
      <div
        className="vertex"
        id="1_11"
      />
      <div
        className="vertex"
        id="1_12"
      />
      <div
        className="vertex"
        id="1_13"
      />
      <div
        className="vertex"
        id="1_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="2_0"
      />
      <div
        className="vertex"
        id="2_1"
      />
      <div
        className="vertex"
        id="2_2"
      />
      <div
        className="vertex"
        id="2_3"
      />
      <div
        className="vertex"
        id="2_4"
      />
      <div
        className="vertex"
        id="2_5"
      />
      <div
        className="vertex"
        id="2_6"
      />
      <div
        className="vertex"
        id="2_7"
      />
      <div
        className="vertex"
        id="2_8"
      />
      <div
        className="vertex"
        id="2_9"
      />
      <div
        className="vertex"
        id="2_10"
      />
      <div
        className="vertex"
        id="2_11"
      />
      <div
        className="vertex"
        id="2_12"
      />
      <div
        className="vertex"
        id="2_13"
      />
      <div
        className="vertex"
        id="2_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="3_0"
      />
      <div
        className="vertex"
        id="3_1"
      />
      <div
        className="vertex"
        id="3_2"
      />
      <div
        className="vertex"
        id="3_3"
      />
      <div
        className="vertex"
        id="3_4"
      />
      <div
        className="vertex"
        id="3_5"
      />
      <div
        className="vertex"
        id="3_6"
      />
      <div
        className="vertex"
        id="3_7"
      />
      <div
        className="vertex"
        id="3_8"
      />
      <div
        className="vertex"
        id="3_9"
      />
      <div
        className="vertex"
        id="3_10"
      />
      <div
        className="vertex"
        id="3_11"
      />
      <div
        className="vertex"
        id="3_12"
      />
      <div
        className="vertex"
        id="3_13"
      />
      <div
        className="vertex"
        id="3_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="4_0"
      />
      <div
        className="vertex"
        id="4_1"
      />
      <div
        className="vertex"
        id="4_2"
      />
      <div
        className="vertex"
        id="4_3"
      />
      <div
        className="vertex"
        id="4_4"
      />
      <div
        className="vertex"
        id="4_5"
      />
      <div
        className="vertex"
        id="4_6"
      />
      <div
        className="vertex"
        id="4_7"
      />
      <div
        className="vertex"
        id="4_8"
      />
      <div
        className="vertex"
        id="4_9"
      />
      <div
        className="vertex"
        id="4_10"
      />
      <div
        className="vertex"
        id="4_11"
      />
      <div
        className="vertex"
        id="4_12"
      />
      <div
        className="vertex"
        id="4_13"
      />
      <div
        className="vertex"
        id="4_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="5_0"
      />
      <div
        className="vertex"
        id="5_1"
      />
      <div
        className="vertex"
        id="5_2"
      />
      <div
        className="vertex"
        id="5_3"
      />
      <div
        className="vertex"
        id="5_4"
      />
      <div
        className="endVertex"
        id="5_5"
      />
      <div
        className="vertex"
        id="5_6"
      />
      <div
        className="vertex"
        id="5_7"
      />
      <div
        className="vertex"
        id="5_8"
      />
      <div
        className="vertex"
        id="5_9"
      />
      <div
        className="vertex"
        id="5_10"
      />
      <div
        className="vertex"
        id="5_11"
      />
      <div
        className="vertex"
        id="5_12"
      />
      <div
        className="vertex"
        id="5_13"
      />
      <div
        className="vertex"
        id="5_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="6_0"
      />
      <div
        className="vertex"
        id="6_1"
      />
      <div
        className="vertex"
        id="6_2"
      />
      <div
        className="vertex"
        id="6_3"
      />
      <div
        className="vertex"
        id="6_4"
      />
      <div
        className="vertex"
        id="6_5"
      />
      <div
        className="vertex"
        id="6_6"
      />
      <div
        className="vertex"
        id="6_7"
      />
      <div
        className="vertex"
        id="6_8"
      />
      <div
        className="vertex"
        id="6_9"
      />
      <div
        className="vertex"
        id="6_10"
      />
      <div
        className="vertex"
        id="6_11"
      />
      <div
        className="vertex"
        id="6_12"
      />
      <div
        className="vertex"
        id="6_13"
      />
      <div
        className="vertex"
        id="6_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="7_0"
      />
      <div
        className="vertex"
        id="7_1"
      />
      <div
        className="vertex"
        id="7_2"
      />
      <div
        className="vertex"
        id="7_3"
      />
      <div
        className="vertex"
        id="7_4"
      />
      <div
        className="vertex"
        id="7_5"
      />
      <div
        className="vertex"
        id="7_6"
      />
      <div
        className="vertex"
        id="7_7"
      />
      <div
        className="vertex"
        id="7_8"
      />
      <div
        className="vertex"
        id="7_9"
      />
      <div
        className="vertex"
        id="7_10"
      />
      <div
        className="vertex"
        id="7_11"
      />
      <div
        className="vertex"
        id="7_12"
      />
      <div
        className="vertex"
        id="7_13"
      />
      <div
        className="vertex"
        id="7_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="8_0"
      />
      <div
        className="vertex"
        id="8_1"
      />
      <div
        className="vertex"
        id="8_2"
      />
      <div
        className="vertex"
        id="8_3"
      />
      <div
        className="vertex"
        id="8_4"
      />
      <div
        className="vertex"
        id="8_5"
      />
      <div
        className="vertex"
        id="8_6"
      />
      <div
        className="vertex"
        id="8_7"
      />
      <div
        className="vertex"
        id="8_8"
      />
      <div
        className="vertex"
        id="8_9"
      />
      <div
        className="vertex"
        id="8_10"
      />
      <div
        className="vertex"
        id="8_11"
      />
      <div
        className="vertex"
        id="8_12"
      />
      <div
        className="vertex"
        id="8_13"
      />
      <div
        className="vertex"
        id="8_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="9_0"
      />
      <div
        className="vertex"
        id="9_1"
      />
      <div
        className="vertex"
        id="9_2"
      />
      <div
        className="vertex"
        id="9_3"
      />
      <div
        className="vertex"
        id="9_4"
      />
      <div
        className="vertex"
        id="9_5"
      />
      <div
        className="vertex"
        id="9_6"
      />
      <div
        className="vertex"
        id="9_7"
      />
      <div
        className="vertex"
        id="9_8"
      />
      <div
        className="vertex"
        id="9_9"
      />
      <div
        className="vertex"
        id="9_10"
      />
      <div
        className="vertex"
        id="9_11"
      />
      <div
        className="vertex"
        id="9_12"
      />
      <div
        className="vertex"
        id="9_13"
      />
      <div
        className="vertex"
        id="9_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="10_0"
      />
      <div
        className="vertex"
        id="10_1"
      />
      <div
        className="vertex"
        id="10_2"
      />
      <div
        className="vertex"
        id="10_3"
      />
      <div
        className="vertex"
        id="10_4"
      />
      <div
        className="vertex"
        id="10_5"
      />
      <div
        className="vertex"
        id="10_6"
      />
      <div
        className="vertex"
        id="10_7"
      />
      <div
        className="vertex"
        id="10_8"
      />
      <div
        className="vertex"
        id="10_9"
      />
      <div
        className="vertex"
        id="10_10"
      />
      <div
        className="vertex"
        id="10_11"
      />
      <div
        className="vertex"
        id="10_12"
      />
      <div
        className="vertex"
        id="10_13"
      />
      <div
        className="vertex"
        id="10_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="11_0"
      />
      <div
        className="vertex"
        id="11_1"
      />
      <div
        className="vertex"
        id="11_2"
      />
      <div
        className="vertex"
        id="11_3"
      />
      <div
        className="vertex"
        id="11_4"
      />
      <div
        className="vertex"
        id="11_5"
      />
      <div
        className="vertex"
        id="11_6"
      />
      <div
        className="vertex"
        id="11_7"
      />
      <div
        className="vertex"
        id="11_8"
      />
      <div
        className="vertex"
        id="11_9"
      />
      <div
        className="vertex"
        id="11_10"
      />
      <div
        className="vertex"
        id="11_11"
      />
      <div
        className="vertex"
        id="11_12"
      />
      <div
        className="vertex"
        id="11_13"
      />
      <div
        className="vertex"
        id="11_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="12_0"
      />
      <div
        className="vertex"
        id="12_1"
      />
      <div
        className="vertex"
        id="12_2"
      />
      <div
        className="vertex"
        id="12_3"
      />
      <div
        className="vertex"
        id="12_4"
      />
      <div
        className="vertex"
        id="12_5"
      />
      <div
        className="vertex"
        id="12_6"
      />
      <div
        className="vertex"
        id="12_7"
      />
      <div
        className="vertex"
        id="12_8"
      />
      <div
        className="vertex"
        id="12_9"
      />
      <div
        className="vertex"
        id="12_10"
      />
      <div
        className="vertex"
        id="12_11"
      />
      <div
        className="vertex"
        id="12_12"
      />
      <div
        className="vertex"
        id="12_13"
      />
      <div
        className="vertex"
        id="12_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="13_0"
      />
      <div
        className="vertex"
        id="13_1"
      />
      <div
        className="vertex"
        id="13_2"
      />
      <div
        className="vertex"
        id="13_3"
      />
      <div
        className="vertex"
        id="13_4"
      />
      <div
        className="vertex"
        id="13_5"
      />
      <div
        className="vertex"
        id="13_6"
      />
      <div
        className="vertex"
        id="13_7"
      />
      <div
        className="vertex"
        id="13_8"
      />
      <div
        className="vertex"
        id="13_9"
      />
      <div
        className="vertex"
        id="13_10"
      />
      <div
        className="vertex"
        id="13_11"
      />
      <div
        className="vertex"
        id="13_12"
      />
      <div
        className="vertex"
        id="13_13"
      />
      <div
        className="vertex"
        id="13_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="14_0"
      />
      <div
        className="vertex"
        id="14_1"
      />
      <div
        className="vertex"
        id="14_2"
      />
      <div
        className="vertex"
        id="14_3"
      />
      <div
        className="vertex"
        id="14_4"
      />
      <div
        className="vertex"
        id="14_5"
      />
      <div
        className="vertex"
        id="14_6"
      />
      <div
        className="vertex"
        id="14_7"
      />
      <div
        className="vertex"
        id="14_8"
      />
      <div
        className="vertex"
        id="14_9"
      />
      <div
        className="vertex"
        id="14_10"
      />
      <div
        className="vertex"
        id="14_11"
      />
      <div
        className="vertex"
        id="14_12"
      />
      <div
        className="vertex"
        id="14_13"
      />
      <div
        className="vertex"
        id="14_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="15_0"
      />
      <div
        className="vertex"
        id="15_1"
      />
      <div
        className="vertex"
        id="15_2"
      />
      <div
        className="vertex"
        id="15_3"
      />
      <div
        className="vertex"
        id="15_4"
      />
      <div
        className="vertex"
        id="15_5"
      />
      <div
        className="vertex"
        id="15_6"
      />
      <div
        className="vertex"
        id="15_7"
      />
      <div
        className="vertex"
        id="15_8"
      />
      <div
        className="vertex"
        id="15_9"
      />
      <div
        className="vertex"
        id="15_10"
      />
      <div
        className="vertex"
        id="15_11"
      />
      <div
        className="vertex"
        id="15_12"
      />
      <div
        className="vertex"
        id="15_13"
      />
      <div
        className="vertex"
        id="15_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="16_0"
      />
      <div
        className="vertex"
        id="16_1"
      />
      <div
        className="vertex"
        id="16_2"
      />
      <div
        className="vertex"
        id="16_3"
      />
      <div
        className="vertex"
        id="16_4"
      />
      <div
        className="vertex"
        id="16_5"
      />
      <div
        className="vertex"
        id="16_6"
      />
      <div
        className="vertex"
        id="16_7"
      />
      <div
        className="vertex"
        id="16_8"
      />
      <div
        className="vertex"
        id="16_9"
      />
      <div
        className="vertex"
        id="16_10"
      />
      <div
        className="vertex"
        id="16_11"
      />
      <div
        className="vertex"
        id="16_12"
      />
      <div
        className="vertex"
        id="16_13"
      />
      <div
        className="vertex"
        id="16_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="17_0"
      />
      <div
        className="vertex"
        id="17_1"
      />
      <div
        className="vertex"
        id="17_2"
      />
      <div
        className="vertex"
        id="17_3"
      />
      <div
        className="vertex"
        id="17_4"
      />
      <div
        className="vertex"
        id="17_5"
      />
      <div
        className="vertex"
        id="17_6"
      />
      <div
        className="vertex"
        id="17_7"
      />
      <div
        className="vertex"
        id="17_8"
      />
      <div
        className="vertex"
        id="17_9"
      />
      <div
        className="vertex"
        id="17_10"
      />
      <div
        className="vertex"
        id="17_11"
      />
      <div
        className="vertex"
        id="17_12"
      />
      <div
        className="vertex"
        id="17_13"
      />
      <div
        className="vertex"
        id="17_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="18_0"
      />
      <div
        className="vertex"
        id="18_1"
      />
      <div
        className="vertex"
        id="18_2"
      />
      <div
        className="vertex"
        id="18_3"
      />
      <div
        className="vertex"
        id="18_4"
      />
      <div
        className="vertex"
        id="18_5"
      />
      <div
        className="vertex"
        id="18_6"
      />
      <div
        className="vertex"
        id="18_7"
      />
      <div
        className="vertex"
        id="18_8"
      />
      <div
        className="vertex"
        id="18_9"
      />
      <div
        className="vertex"
        id="18_10"
      />
      <div
        className="vertex"
        id="18_11"
      />
      <div
        className="vertex"
        id="18_12"
      />
      <div
        className="vertex"
        id="18_13"
      />
      <div
        className="vertex"
        id="18_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="19_0"
      />
      <div
        className="vertex"
        id="19_1"
      />
      <div
        className="vertex"
        id="19_2"
      />
      <div
        className="vertex"
        id="19_3"
      />
      <div
        className="vertex"
        id="19_4"
      />
      <div
        className="vertex"
        id="19_5"
      />
      <div
        className="vertex"
        id="19_6"
      />
      <div
        className="vertex"
        id="19_7"
      />
      <div
        className="vertex"
        id="19_8"
      />
      <div
        className="vertex"
        id="19_9"
      />
      <div
        className="vertex"
        id="19_10"
      />
      <div
        className="vertex"
        id="19_11"
      />
      <div
        className="vertex"
        id="19_12"
      />
      <div
        className="vertex"
        id="19_13"
      />
      <div
        className="vertex"
        id="19_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="20_0"
      />
      <div
        className="vertex"
        id="20_1"
      />
      <div
        className="vertex"
        id="20_2"
      />
      <div
        className="vertex"
        id="20_3"
      />
      <div
        className="vertex"
        id="20_4"
      />
      <div
        className="vertex"
        id="20_5"
      />
      <div
        className="vertex"
        id="20_6"
      />
      <div
        className="vertex"
        id="20_7"
      />
      <div
        className="vertex"
        id="20_8"
      />
      <div
        className="vertex"
        id="20_9"
      />
      <div
        className="vertex"
        id="20_10"
      />
      <div
        className="vertex"
        id="20_11"
      />
      <div
        className="vertex"
        id="20_12"
      />
      <div
        className="vertex"
        id="20_13"
      />
      <div
        className="vertex"
        id="20_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="21_0"
      />
      <div
        className="vertex"
        id="21_1"
      />
      <div
        className="vertex"
        id="21_2"
      />
      <div
        className="vertex"
        id="21_3"
      />
      <div
        className="vertex"
        id="21_4"
      />
      <div
        className="vertex"
        id="21_5"
      />
      <div
        className="vertex"
        id="21_6"
      />
      <div
        className="vertex"
        id="21_7"
      />
      <div
        className="vertex"
        id="21_8"
      />
      <div
        className="vertex"
        id="21_9"
      />
      <div
        className="vertex"
        id="21_10"
      />
      <div
        className="vertex"
        id="21_11"
      />
      <div
        className="vertex"
        id="21_12"
      />
      <div
        className="vertex"
        id="21_13"
      />
      <div
        className="vertex"
        id="21_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="22_0"
      />
      <div
        className="vertex"
        id="22_1"
      />
      <div
        className="vertex"
        id="22_2"
      />
      <div
        className="vertex"
        id="22_3"
      />
      <div
        className="vertex"
        id="22_4"
      />
      <div
        className="vertex"
        id="22_5"
      />
      <div
        className="vertex"
        id="22_6"
      />
      <div
        className="vertex"
        id="22_7"
      />
      <div
        className="vertex"
        id="22_8"
      />
      <div
        className="vertex"
        id="22_9"
      />
      <div
        className="vertex"
        id="22_10"
      />
      <div
        className="vertex"
        id="22_11"
      />
      <div
        className="vertex"
        id="22_12"
      />
      <div
        className="vertex"
        id="22_13"
      />
      <div
        className="vertex"
        id="22_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="23_0"
      />
      <div
        className="vertex"
        id="23_1"
      />
      <div
        className="vertex"
        id="23_2"
      />
      <div
        className="vertex"
        id="23_3"
      />
      <div
        className="vertex"
        id="23_4"
      />
      <div
        className="vertex"
        id="23_5"
      />
      <div
        className="vertex"
        id="23_6"
      />
      <div
        className="vertex"
        id="23_7"
      />
      <div
        className="vertex"
        id="23_8"
      />
      <div
        className="vertex"
        id="23_9"
      />
      <div
        className="vertex"
        id="23_10"
      />
      <div
        className="vertex"
        id="23_11"
      />
      <div
        className="vertex"
        id="23_12"
      />
      <div
        className="vertex"
        id="23_13"
      />
      <div
        className="vertex"
        id="23_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="24_0"
      />
      <div
        className="vertex"
        id="24_1"
      />
      <div
        className="vertex"
        id="24_2"
      />
      <div
        className="vertex"
        id="24_3"
      />
      <div
        className="vertex"
        id="24_4"
      />
      <div
        className="vertex"
        id="24_5"
      />
      <div
        className="vertex"
        id="24_6"
      />
      <div
        className="vertex"
        id="24_7"
      />
      <div
        className="vertex"
        id="24_8"
      />
      <div
        className="vertex"
        id="24_9"
      />
      <div
        className="vertex"
        id="24_10"
      />
      <div
        className="vertex"
        id="24_11"
      />
      <div
        className="vertex"
        id="24_12"
      />
      <div
        className="vertex"
        id="24_13"
      />
      <div
        className="vertex"
        id="24_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="25_0"
      />
      <div
        className="vertex"
        id="25_1"
      />
      <div
        className="vertex"
        id="25_2"
      />
      <div
        className="vertex"
        id="25_3"
      />
      <div
        className="vertex"
        id="25_4"
      />
      <div
        className="vertex"
        id="25_5"
      />
      <div
        className="vertex"
        id="25_6"
      />
      <div
        className="vertex"
        id="25_7"
      />
      <div
        className="vertex"
        id="25_8"
      />
      <div
        className="vertex"
        id="25_9"
      />
      <div
        className="vertex"
        id="25_10"
      />
      <div
        className="vertex"
        id="25_11"
      />
      <div
        className="vertex"
        id="25_12"
      />
      <div
        className="vertex"
        id="25_13"
      />
      <div
        className="vertex"
        id="25_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="26_0"
      />
      <div
        className="vertex"
        id="26_1"
      />
      <div
        className="vertex"
        id="26_2"
      />
      <div
        className="vertex"
        id="26_3"
      />
      <div
        className="vertex"
        id="26_4"
      />
      <div
        className="vertex"
        id="26_5"
      />
      <div
        className="vertex"
        id="26_6"
      />
      <div
        className="vertex"
        id="26_7"
      />
      <div
        className="vertex"
        id="26_8"
      />
      <div
        className="vertex"
        id="26_9"
      />
      <div
        className="vertex"
        id="26_10"
      />
      <div
        className="vertex"
        id="26_11"
      />
      <div
        className="vertex"
        id="26_12"
      />
      <div
        className="vertex"
        id="26_13"
      />
      <div
        className="vertex"
        id="26_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="27_0"
      />
      <div
        className="vertex"
        id="27_1"
      />
      <div
        className="vertex"
        id="27_2"
      />
      <div
        className="vertex"
        id="27_3"
      />
      <div
        className="vertex"
        id="27_4"
      />
      <div
        className="vertex"
        id="27_5"
      />
      <div
        className="vertex"
        id="27_6"
      />
      <div
        className="vertex"
        id="27_7"
      />
      <div
        className="vertex"
        id="27_8"
      />
      <div
        className="vertex"
        id="27_9"
      />
      <div
        className="vertex"
        id="27_10"
      />
      <div
        className="vertex"
        id="27_11"
      />
      <div
        className="vertex"
        id="27_12"
      />
      <div
        className="vertex"
        id="27_13"
      />
      <div
        className="vertex"
        id="27_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="28_0"
      />
      <div
        className="vertex"
        id="28_1"
      />
      <div
        className="vertex"
        id="28_2"
      />
      <div
        className="vertex"
        id="28_3"
      />
      <div
        className="vertex"
        id="28_4"
      />
      <div
        className="vertex"
        id="28_5"
      />
      <div
        className="vertex"
        id="28_6"
      />
      <div
        className="vertex"
        id="28_7"
      />
      <div
        className="vertex"
        id="28_8"
      />
      <div
        className="vertex"
        id="28_9"
      />
      <div
        className="vertex"
        id="28_10"
      />
      <div
        className="vertex"
        id="28_11"
      />
      <div
        className="vertex"
        id="28_12"
      />
      <div
        className="vertex"
        id="28_13"
      />
      <div
        className="vertex"
        id="28_14"
      />
    </div>
    <div>
      <div
        className="vertex"
        id="29_0"
      />
      <div
        className="vertex"
        id="29_1"
      />
      <div
        className="vertex"
        id="29_2"
      />
      <div
        className="vertex"
        id="29_3"
      />
      <div
        className="vertex"
        id="29_4"
      />
      <div
        className="vertex"
        id="29_5"
      />
      <div
        className="vertex"
        id="29_6"
      />
      <div
        className="vertex"
        id="29_7"
      />
      <div
        className="vertex"
        id="29_8"
      />
      <div
        className="vertex"
        id="29_9"
      />
      <div
        className="vertex"
        id="29_10"
      />
      <div
        className="vertex"
        id="29_11"
      />
      <div
        className="vertex"
        id="29_12"
      />
      <div
        className="vertex"
        id="29_13"
      />
      <div
        className="vertex"
        id="29_14"
      />
    </div>
  </div>
</div>
`;
